model/modeldecoder/v2/model.go

```diff
@@ -106,6 +106,9 @@ type contextMessage struct {
 	Headers nullable.HTTPHeader `json:"headers"`
 	// Queue holds information about the message queue where the message is received.
 	Queue contextMessageQueue `json:"queue"`
+	// RoutingKey holds the optional routing key of the received message as set
+	// on the queuing system. Currently only used on for RabbitMQ messages.
```

Comment:
I'm not entirely convinced about the last sentence about RabbitMQ. Removing it might be better than tying this to a specific queuing technology. It's only referenced in the description of the field, but still...
```suggestion
	// RoutingKey holds the optional routing key of the received message as set
	// on the queuing system.
```

Reply:
Maybe reword it as an example? e.g.

> // RoutingKey holds the optional routing key of the received message as set
> // on the queuing system, such as in RabbitMQ.